### DISCUSSION TITLE
Copilot Enterprises

### DISCUSSION THREAD

**Discussion by ChaudharyAnuj** [community/community#46184] [Enterprise] [answered]
### Select Topic Area

Question

### Body

I had question regarding Copilot Enterprises edition. if Copilot enterprises licence is bought 

1. Do copilot will trained  on organisations private repo ?
2. Do copilot will include organisations repos while generating the suggestion ?

**Comment by davecheney** [comment, marked as answer]
> Do copilot will trained on organisations private repo ?

No.

> Do copilot will include organisations repos while generating the suggestion ?

No.
